plugins/Eq/EqCurve.cpp

```diff
@@ -149,7 +150,8 @@ void EqHandle::paint( QPainter *painter, const QStyleOptionGraphicsItem *option,
 		}
 
 		QFont painterFont = painter->font();
-		painterFont.setPointSizeF( painterFont.pointSizeF() * 0.7 );
+		int newFont = static_cast<int>(painterFont.pointSizeF());
+		adjustedToPixelSize(painterFont, newFont); // shrink the font
```

Comment:
Why is the pixel size of the font set via a point size that has been cast? I think it would be better to set it directly to a matching corresponding pixel size here.

Reply:
Even if it worked, it just looks weird because the contexts of point sizes and pixel sizes are mixed.

However, I also noticed that your code does not work anyway because the method `adjustedToPixelSize` returns the adjusted font which goes nowhere in your code. I found that the following works and looks good:
```
painter->setFont(adjustedToPixelSize(painter->font(), 12));
```
Which should be the equivalent to the following in your branch:
```
painter->setFont(adjustedToPixelSize(painter->font(), SMALL_FONT_SIZE));
```

The above replaces the whole code block:
```
QFont painterFont = painter->font();
int newFont = static_cast<int>(painterFont.pointSizeF());
adjustedToPixelSize(painterFont, newFont); // shrink the font
painter->setFont( painterFont );
```

Reply:
Yeah missed the "adjusted font goes nowhere" part. I'll address that too and your solution, especially the second line where you used the `SMALL_FONT_SIZE` macro looks good enough. 

I'll implement soon.

Reply:
done with https://github.com/LMMS/lmms/pull/7493/commits/6667544bc65ecc0bda271a55b191b5e2459a368e